hare/messagevalidation.go

```diff
@@ -32,9 +34,17 @@ func (ev *eligibilityValidator) validateRole(m *Msg) bool {
 	// TODO: validate role proof sig
 
 	pub := m.PubKey
+	layer := types.LayerID(m.InnerMsg.InstanceId)
+	nId := types.NodeId{Key: pub.String()}
```

Comment:
you're not setting `VRFPublicKey`.. Is it in purpose?

Reply:
Fixed